slices/libproc2-0.yaml

```
@@ -0,0 +1,15 @@
+package: libproc2-0
+
+essential:
+  - libproc2-0_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libsystemd0_libs
+    contents:
+      /usr/lib/*-linux-*/libproc2.so.0:
```

```suggestion
      /usr/lib/*-linux-*/libproc2.so.0*:
```
Otherwise, you would have a broken symlink.